cli/cmd/plugin/standalone-cluster/main.go

```diff
@@ -15,6 +15,11 @@ var descriptor = cliv1alpha1.PluginDescriptor{
 	Name:        "standalone-cluster",
 	Description: "Create clusters without a dedicated management cluster",
 	Group:       cliv1alpha1.RunCmdGroup,
+
+	// Since standalone cluster is being deprecated, no changes are being made to it's dependencies,
+	// and cli.BuildVersion was deprecated in the build version of tanzu framework v0.10.0,
+	// we need to manually set the plugin version in order to compile
+	Version: "v0.9.0",
```

Comment:
Note that we're calling out an explicit version here. This means that running `make release` locally fails since it can't find the `0.9.2-dev...` release.

This was necessary since we won't be bumping the tanzu-framework version for standalone-cluster but we still need to be able to build with `hack/builder` which is injecting the `buildinfo.Version`. So otherwise, we end up with the "version cannot be empty" problem that we run into from time to time.

To get around this, I propose we keep this as a literal string, change it as needed when releasing, and then eventually it will be fully deprecated and we won't have to mess with changing the version of tanzu framework we use in standalone cluster

Reply:
Yea, this is totally fine. I would suggest keeping the last release version `v0.9.1` for the value and then we just need then to hardcore the version for just this plugin in the package script which needs to be done in 3 places (here is the first place https://github.com/vmware-tanzu/community-edition/blob/main/hack/release/package-release.sh#L68).

Reply:
Which other locations were needed for an update here? I updated the link you mentioned and ran `make release` without problem and the tarball looks good

Reply:
just that file in those 3 places. yup, it looks good!

Reply:
I forgot... we have a loop now for all 3. just in the one place!